frame/balances/src/tests.rs

```diff
@@ -153,15 +151,15 @@ macro_rules! decl_tests {
 					assert!(<ChargeTransactionPayment<$test> as SignedExtension>::pre_dispatch(
 						ChargeTransactionPayment::from(1),
 						&1,
-						CALL,
-						info_from_weight(1),
+						&Default::default(),
```

Comment:
To be honest I preferred previous way. At least we knew that this is a `Call` we are passing here.

Reply:
+1

Reply:
Fixed.